README.md

```diff
@@ -49,19 +37,62 @@ There are a variety of options that customize your Rails application.
 --css=postcss           # Use the PostCSS framework for bundling CSS
 --template=template.rb  # Add additional configuration from template.rb
 --database=postgresql   # Use a PostgreSQL database
+--skip-rubocop          # Skip rubocop integration in favor of Standard Ruby
+--skip-ci               # Skip github actions in favor of our CI generators
 ```
 
-#### Customizing the installation
+If you are using Hotwire, then `--skip-hotwire` and `--skip-action-cable` are automatically removed from this list, as they are required for the Hotwire functionality.
+</details>
+<br />
+
+Add the following options at the end of your `rails_template_18f new` command to overwrite any of those defaults.
 
 | Option | Description |
 |--------|-------------|
 | `--no-skip-<framework>` | Each of the skipped frameworks listed above (also in `railsrc`) can be overridden on the command line. For example: `--no-skip-active-storage` will include support for `ActiveStorage` document uploads |
 | `--javascript=esbuild` | Use [esbuild](https://esbuild.github.io/) instead of [webpack](https://webpack.js.org/) for JavaScript bundling. Note that maintaining IE11 support with esbuild may be tricky. |
 
-You probably won't want to customize the template — that defeats the purpose of using this gem!
-
 _TODO: Documentation on whether you can override the `css` and `database` options._
 
+**Important:** Do not use flags `--skip-bundle` or `--skip-javascript`, or various parts of this template will break.
+
+#### Create your application
+
+<details><summary>If you are using Hotwire, run:</summary>
+
+```
+$ rails_template_18f new <project name> --hotwire ADDITIONAL_CONFIG_OPTIONS
```

Comment:
I don't love `ADDITIONAL_CONFIG_OPTIONS` but couldn't think of a better way to say "insert any --no-skip... options here"